C#/Botkit/Microsoft.BotKit.Adapters.Slack/SlackAdapter.cs

```diff
@@ -556,8 +568,10 @@ private bool VerifySignatureAsync(HttpRequestMessage request, HttpResponseMessag
 
                 string baseString = String.Join(":", signature);
```

Comment:
_Signature_ must be a _string array_. You can use both _toString_ methods in _timestamp_ & _body_ objects